client/js/store/statement/Statement.js

```diff
@@ -757,7 +751,6 @@ export default {
           Accept: 'application/vnd.api+json'
         }
       })
-        .then(this.api.checkResponse)
         .then(response => {
           let assignee = {}
           if (assigneeId === '' || assigneeId == null) {
```

Comment:
Shouldn't it be
```
assignee = { id: response.data.data.id, uId: response.data.data.id, name: response.data.data.attributes.name, orgaName: response.data.data.attributes.orgaName }
```
in the else block below?

Reply:
thanks for spotting f67ecc730d8004187e520ddf420c2fadc5277058